tree/treeplayer/inc/ROOT/TDataFrameInterface.hxx

```diff
@@ -958,6 +959,7 @@ protected:
       return BranchNames_t(bnBegin, bnBegin + nExpectedBranches);
    }
 
+#if 0
```

Comment:
This is WIP, as mentioned in the header of the PR. I want to be able to switch between the old and new implementations for testing. It will, of course, go away before this gets merged.